src/ape/managers/networks.py

```diff
@@ -119,6 +120,36 @@ def ecosystems(self) -> Dict[str, EcosystemAPI]:
         self._ecosystems_by_project[project_name] = ecosystem_dict
         return ecosystem_dict
 
+    def create_adhoc_geth_provider(self, uri: str) -> ProviderAPI:
+        """
+        Create an ad-hoc connection to a URI using the GethProvider core plugin.
+        **NOTE**: This provider will assume EVM-like provider and this is generally not recommended.
+        Use plugins when possible!
+
+        Args:
+            uri (str): The URI of the node.
+
+        Returns:
+            :class:`~ape.api.providers.ProviderAPI`: The Geth provider implementation that comes with Ape.
+        """
+
+        geth_class = None
+        for plugin_name, (_, _, provider_class) in self.plugin_manager.providers:
+            if plugin_name == "geth":
+                geth_class = provider_class
+                break
+
+        if geth_class is None:
+            raise NetworkError("Core Geth plugin missing.")
```

Comment:
question: Is this possible?  Unless you mod ape directly, is there a case where `ape_geth` could be unavailable?

Reply:
It is possible when you have a messed up editable install of the core ape library.

Reply:
Fair enough, defensive is good.  I'm unable to mark this as resolved, though.